### PR TITLE
zh: Add TidbMonitor Ingress in monitor document

### PR DESCRIPTION
**Pull request by Yisaer**
### What is changed, added or deleted? (Required)
Add TidbMonitor Ingress in the document

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/monitor-using-tidbmonitor.md [hunk 1]
```diff
@@ -171,6 +171,99 @@ spec:
...
+    version: v3.0.5
+  reloader:
+    baseImage: pingcap/tidb-monitor-reloader
+    version: v1.0.1
+  imagePullPolicy: IfNotPresent
+```
+
+你可以通过 `spec.prometheus.ingress.annotations` 与 `spec.grafana.ingress.annotations` 来设置对应的 Ingress Annotations 的设置。如果你使用的是默认的 Nginx Ingress 方案，你可以在 [Nginx Ingress Controller Annotation](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/) 了解更多关于 Annotations 的详情。
+
+`TidbMonitor` 的 Ingress 设置同样支持设置 tls ， 以下是一个设置为 Ingress 设置 tls 的例子。 你可以通过 [Ingress TLS](https://kubernetes.io/zh/docs/concepts/services-networking/ingress/#tls) 来了解更多关于 Ingress TLS 的资料。
```
Comment (ran-huang) [inline zh/monitor-using-tidbmonitor.md:218]:
```suggestion
`TidbMonitor` 的 Ingress 设置同样支持设置 TLS，以下是一个为 Ingress 设置 TLS 的例子。你可以通过 [Ingress TLS](https://kubernetes.io/zh/docs/concepts/services-networking/ingress/#tls) 来了解更多关于 Ingress TLS 的资料。
```

File: zh/monitor-using-tidbmonitor.md [hunk 2]
```diff
@@ -171,6 +171,99 @@ spec:
...
+```
+
+ TLS Secret 必须包含名为 tls.crt 和 tls.key 的密钥，这些密钥包含用于 TLS 的证书和私钥，例如：
+
+ ```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: testsecret-tls
+  namespace: default
```
Comment (DanielZhangQD) [inline zh/monitor-using-tidbmonitor.md:260]: Remove the namespace here? There is no namespace in the above yam examples.

File: zh/monitor-using-tidbmonitor.md [hunk 3]
```diff
@@ -171,6 +171,99 @@ spec:
   imagePullPolicy: IfNotPresent
 ```
 
+## 开启 Ingress
+
+目前, `TidbMonitor` 提供了通过 Ingress 将 Prometheus/Grafana 服务暴露出去的方式，你可以通过[Ingress](https://kubernetes.io/zh/docs/concepts/services-networking/ingress/)了解更多关于 Ingress 的详情。
```
Comment (DanielZhangQD) [inline zh/monitor-using-tidbmonitor.md:176]: Should we also mention that users need to deploy ingress controller by themselves?

Comment (Yisaer) [reply zh/monitor-using-tidbmonitor.md:184]: updated.

File: zh/monitor-using-tidbmonitor.md (hunk 2 above)
Comment (Yisaer) [reply zh/monitor-using-tidbmonitor.md]: updated.

File: zh/monitor-using-tidbmonitor.md [hunk 4]
```diff
@@ -171,6 +171,107 @@ spec:
...
+
+### 环境准备
+
+使用 `Ingress` 前需要 Kubernetes 集群安装有 `Ingress` 控制器，仅创建 `Ingress` 资源无效。您可能需要部署 `Ingress` 控制器，例如 [ingress-nginx](https://kubernetes.github.io/ingress-nginx/deploy/)。您可以从许多 [Ingress控制器](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) 中进行选择。
+
+更多关于 `Ingress` 环境准备，可以参考 [Ingress 环境准备](https://kubernetes.io/zh/docs/concepts/services-networking/ingress/#%E7%8E%AF%E5%A2%83%E5%87%86%E5%A4%87)
+
+### 使用 Ingress 访问 TidbMonitor 
+
+目前, `TidbMonitor` 提供了通过 Ingress 将 Prometheus/Grafana 服务暴露出去的方式，你可以通过[Ingress](https://kubernetes.io/zh/docs/concepts/services-networking/ingress/)了解更多关于 Ingress 的详情。
```
Comment (ran-huang) [inline zh/monitor-using-tidbmonitor.md:184]:
```suggestion
目前, `TidbMonitor` 提供了通过 Ingress 将 Prometheus/Grafana 服务暴露出去的方式，你可以通过 [Ingress 文档](https://kubernetes.io/zh/docs/concepts/services-networking/ingress/)了解更多关于 Ingress 的详情。
```

File: zh/monitor-using-tidbmonitor.md [hunk 5]
```diff
@@ -171,6 +171,107 @@ spec:
...
+
+以下是一个开启了 Prometheus 与 Grafana Ingress 的 `TidbMonitor` 例子。
```
Comment (ran-huang) [inline zh/monitor-using-tidbmonitor.md:186]:
```suggestion
以下是一个开启了 Prometheus 与 Grafana Ingress 的 `TidbMonitor` 例子：
```

**Pull request merged**